wandb/board/ui/src/components/RunFeed.js

```diff
@@ -264,10 +261,9 @@ class RunFeed extends PureComponent {
   }
 
   render() {
-    let lastDay = 0,
-      stats =
+    let /*stats =
         this.props.project &&
-        Object.keys(JSONparseNaN(this.props.project.summaryMetrics)).sort(),
+        Object.keys(JSONparseNaN(this.props.project.summaryMetrics)).sort(),*/
       runsLength = this.props.runs && this.props.runs.length,
       startIndex = (this.props.currentPage - 1) * this.props.limit,
       endIndex = Math.min(startIndex + this.props.limit, runsLength),
```

Comment:
Why did we comment this and the JSONparseNaN out?

Reply:
Stats variable is not being used, thus JSONparseNaN module will not be in use neither. 
Didn't want to remove that part of the code just yet, as it will maybe be of any help.